tests/integration/tests.c

```diff
@@ -517,45 +479,47 @@ static void test_graph_firewall_intense(void)
 
 static void test_graph_firewall_intense_multiple(void)
 {
-	struct pg_brick *nic, *vtep, *print;
 	struct pg_error *error = NULL;
-	struct ether_addr mac_vtep = {{0xb0, 0xb1, 0xb2,
-				       0xb3, 0xb4, 0xb5} };
-	struct ether_addr mac1 = {{0x52, 0x54, 0x00, 0x12, 0x34, 0x11} };
 	struct branch branches[PG_BRANCHES_NB];
+	struct pg_graph *graphs[PG_BRANCHES_NB];
+	GString *tmp = g_string_new(NULL);
 	int ret = -1;
-	GList *brick_gc = NULL;
 
 	pg_vhost_start("/tmp", &error);
 	check_error(error);
 
-	nic = pg_nic_new_by_id("nic", ring_port(), &error);
-	check_error(error);
-	vtep = pg_vtep_new("vt", PG_BRANCHES_NB, PG_WEST_SIDE,
-			   0x000000EE, mac_vtep, PG_VTEP_DST_PORT,
-			   PG_VTEP_ALL_OPTI, &error);
-	check_error(error);
-
-	print = pg_print_new("main-print", NULL, PG_PRINT_FLAG_MAX,
-			     NULL, &error);
-	check_error(error);
-	PG_GC_CHAINED_ADD(brick_gc, nic, vtep, print);
-
-	pg_brick_chained_links(&error, nic, print, vtep);
-	check_error(error);
 
 	for (int i = 0; i < 100; ++i) {
 		for (int j = 0; j < PG_BRANCHES_NB; j++) {
-			g_assert(add_graph_branch(&branches[j], j, mac1, 0, 0));
```

Comment:
I'm pretty sure you can use add_graph_branch here, it's link_graph_branch that doesn't work with graph.